pkg/skaffold/runner/build_deploy.go

```diff
@@ -87,6 +93,7 @@ func (r *SkaffoldRunner) Build(ctx context.Context, out io.Writer, artifacts []*
 	// Make sure all artifacts are redeployed. Not only those that were just built.
 	r.builds = build.MergeWithPreviousBuilds(bRes, r.builds)
 
+	eventV2.TaskSucceeded("Build", r.devIteration)
```

Comment:
use constants for `Build`, `Devloop` etc.

Reply:
done!